Rahulsingh210345.1 (1).md

```diff
@@ -0,0 +1,24 @@
+---
+name: "Rahulsingh210345"
+project: "Smart Plug"
+---
+
+# Smart Plug
+
+## Summary
+I am going to build a smart plug.This device will  allow users to remotely control and schedule the power to an electrical outlet through a smartphone app or voice assistant.this smart plug is way to learn about the inner workings of smart home technology and gain hands-on experience with electrical engineering and programming.
+
+ 
+## Plan
+To create a Smart Plug following  using the specified components, I will follow these steps:
+
+First I will make the Smart plug with remote monitoring (current, voltage, energy), control (automatic, manual), and notification.then I will connect wifi through appliances remotely. No matter where you are, integrate your device with the web server Then I will attach the Espressif ESP8266 ESP-01ESP WiFi Module.Then Set the Schedules/Timer  for turning ON and OFF your devices automatically,saving power and Time.This smart plug helps you monitor your devices energy consumption for the entire day and is suitable for conecting electrical appliances like Geysers,motors for Water Tanks,Heaters,Microwave Ovens,ACs,etc
+
+|[ Product                      | Supplier/Link                         | Cost    |
+| ---------------------------- | ------------------------------------- | ------- |
+| ESP Wifi Module              | https://www.sparkfun.com/products/17146          | $40.00  |
+| AC to DC Converter           |  https://www.amazon.com/Transformer-MKASYON-Switching-Converter-Security/dp/B07GFLFTV2     | $35.06|
+| Measuring Voltage and Current|  https://www.tme.eu/en/details/abx00041/arduino-solutions/arduino/nano-motor-carrier/      | $25.00 |
+| Smart Wi-Fi Plug Mechanical Box |https://jujukart.com/shop/smart-electronics/teckin-smart-wi-fi-plug-with-price-in-nepal/| $75.00 |
```

Comment:
What's the purpose of this part? Seems like an already made consumer product.

Reply:
please, help me then what I do?

Reply:
> What's the purpose of this part? Seems like an already made consumer product.
I have no idea sir plz help me?

Reply:
How can I help? Sorry I'm really lost and just want to know why you've included this product in your purchasing list.